CMakeLists.txt

```diff
@@ -17,6 +17,18 @@ set(CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/CMake" ${CMAKE_MODULE_PATH})
 # set the project name
 project(velox)
 
+set(DEPENDENCY_INSTALL_PREFIX "deps")
+list(APPEND CMAKE_PREFIX_PATH ${DEPENDENCY_INSTALL_PREFIX})
+include_directories(SYSTEM "${DEPENDENCY_INSTALL_PREFIX}/include")
+
+if(${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
+  # Fix link errors on macOS BigSur, without this link fails unable to find
+  # -lSystem.  Based on same workaround found here:
+  # https://github.com/ponylang/ponyc/pull/3686/commits/03cd3f02ffabbc0691382502e4f0579ade52e47c
+  add_link_options(
```

Comment:
Does this problem occur without this change?

Reply:
Yes, on my FB laptop I can't link without this fix
BigSur, 11.6
Darwin mbp 20.6.0 Darwin Kernel Version 20.6.0: Mon Aug 30 06:12:21 PDT 2021; root:xnu-7195.141.6~3/RELEASE_X86_64 x86_64